src/layer/x86/layernorm_x86.h

```diff
@@ -0,0 +1,22 @@
+#ifndef LAYER_LAYERNORM_X86_H
+#define LAYER_LAYERNORM_X86_H
+
+#include "layernorm.h"
+
+namespace ncnn {
+
+class LayerNorm_x86 : virtual public LayerNorm
+{
+public:
+    LayerNorm_x86();
+
+    virtual int forward_inplace(Mat& bottom_top_blob, const Option& opt) const;
+
+protected:
+    NCNN_FORCEINLINE void fast_1d_layer_norm(float* ptr, int elempack, int elemcount, int size) const;
+    NCNN_FORCEINLINE void fast_fmadd_fmadd(float* ptr, float* a, float* b, int elempack, int elemcount, int size) const;
```

Comment:
member function never inlines in practice.
declare static inline functions in cpp. (ref lstm_x86.cpp)

Reply:
Thanks for advising. I've fixed that.